CRM/Report/Form/Contribute/PCP.php

```diff
@@ -168,11 +170,6 @@ public function __construct() {
       'civicrm_contribution' => array(
         'dao' => 'CRM_Contribute_DAO_Contribution',
         'fields' => array(
-          'contribution_id' => array(
-            'name' => 'id',
-            'no_display' => TRUE,
-            'required' => TRUE,
-          ),
           'receive_date' => array(
```

Comment:
I don't know what the bigger impact of this change will be. I had to include it because otherwise the contribution id is added to the GROUP BY clause when ONLY_FULL_GROUP_BY is enabled, and that causes the aggregate functions to fail since we really don't want to ever group by contribution id, we want to group by PCP page id.